BRAD/justchat.py

```
@@ -0,0 +1,130 @@
+from langchain.callbacks.streaming_stdout import StreamingStdOutCallbackHandler
+from langchain.document_loaders import PyPDFLoader
+from langchain.text_splitter import CharacterTextSplitter
+from langchain.chains.question_answering import load_qa_chain
+from langchain.output_parsers import CommaSeparatedListOutputParser
+from semantic_router.layer import RouteLayer
+from langchain.chains import ConversationChain
+from langchain.prompts import PromptTemplate
+from langchain_core.prompts.prompt import PromptTemplate
+from langchain.docstore.document import Document
+from langchain_chroma import Chroma
+from langchain_community.document_loaders import PyPDFLoader
```

Ruff

Redefinition of unused `PyPDFLoader` from line 19

(F811)